3p/facebook.js

```diff
@@ -71,6 +73,38 @@ function getPostContainer(global, data) {
   return container;
 }
 
+/**
+ * Create DOM element for the Facebook embedded content plugin for comments or
+ * comment replies.
+ * @see https://developers.facebook.com/docs/plugins/embedded-comments
+ * @param {!Window} global
+ * @param {!Object} data The element data
+ * @return {!Element} div
+ */
+function getCommentContainer(global, data) {
+  const c = global.document.getElementById('c');
+  const container = global.document.createElement('div');
+  container.setAttribute('data-include-parent', data.includeParent || 'false');
+  container.setAttribute('data-width', c./*OK*/offsetWidth);
+  container.setAttribute('data-href', data.href);
+  container.className = 'fb-comment-embed';
+  return container;
+}
+
+/**
+ * Create DOM element for the Facebook embedded content plugin.
+ * @param {!Window} global
+ * @param {!Object} data The element data
+ * @return {!Element} div
+ */
+function getEmbedContainer(global, data) {
+  if (data.embedType == 'comment') {
+    return getCommentContainer(global, data);
+  }
+
+  return getPostContainer(global, data);
```

Comment:
I feel like now `getPostContainer()` should be split into `getPostContainer()` and `getVideoContainer()` as we add more types. But for later.

Reply:
Sounds good. Split these up into separate functions and did a bit of refactoring for the other containers as well.